dev/local/Makefile

```
@@ -136,8 +140,17 @@ py:
 		$(M1) \
 		-f repos/delphi/delphi-epidata/dev/docker/python/Dockerfile .
 
+.PHONY=redis
+redis:
+	@docker build -t delphi_redis \
+		-f repos/delphi/delphi-epidata/dev/docker/redis/Dockerfile .
+
+	@docker run --rm -d -p 127.0.0.1:6379:6379 \
+		--network delphi-net \
+		--name delphi_redis delphi_redis >$(LOG_WEB) 2>&1 &
```

was this a missed copy-paste or intentional?

```suggestion
		--name delphi_redis delphi_redis >$(LOG_REDIS) 2>&1 &
```